src/queries/profile.js

```diff
@@ -6,13 +6,13 @@ const PROFILE_QUERY = gql`query ($userId: String!) {
       hris {
         values {
           businessTitle,
-          team,
-          entity,
-          locationDescription,
-          timeZone,
-          workerType,
-          wprDeskNumber,
-          costCenter,
+          Team,
+          Entity,
+          LocationDescription,
+          TimeZone,
+          WorkerType,
+          WPRDeskNumber,
+          CostCenter,
         }
```

Comment:
Curious about the reason for this change… all other properties in this file are camelCase and with UpperCamelCase they no longer match GraphQL query responses from the profile faker? (I am likely missing something)

Reply:
@hidde you are correct. HRIS data have a special formatting and they need to be like this. In order to have a consistent name formatting I opened this PR [0] which basically allows us to query GraphQL in a consistent way regardless of the data format (cc @fiji-flo )

[0] https://github.com/mozilla-iam/iam-profile-faker/pull/21

Reply:
This also means that when we merge this other PR, this is no longer needed.